ast/parser.go

```diff
@@ -2309,12 +2309,10 @@ type rawAnnotation struct {
 	Organizations    []string               `yaml:"organizations"`
 	RelatedResources []interface{}          `yaml:"related_resources"`
 	Authors          []interface{}          `yaml:"authors"`
-	Schemas          []rawSchemaAnnotation  `yaml:"schemas"`
+	Schemas          []map[string]any       `yaml:"schemas"`
```

Comment:
I'm not sure why we had the alias, @johanfylling would know. But I'm surprised this needed to be updated.

Reply:
I don't know/recall why we have this type. It was likely for anchoring some method that never made the cut, or typing for typing's sake .. 😄